jykuo-love-shiritori/twp
# Default Image Path to Parrot if empty

Also fix #161




### Changes


- Added more partying

### Testing Approach (if unconventional)


- Create new user
- Go to shop info or user info
- You should see parrot

### Additional context


N/A
